router/state_forward.go

```diff
@@ -106,10 +113,27 @@ func (s *state) _forward(p *peer, f *types.Frame) error {
 			return nil
 		}
 
+	case types.TypeWakeupBroadcast:
+		// Broadcasts are a special case. The _handleBroadcast function will handle
+		// forwarding broadcasts as appropriate.
+		if err := s._handleBroadcast(p, f); err != nil {
+			return fmt.Errorf("s._handleBroadcast (port %d): %w", p.port, err)
+		}
+		return nil
+
 	case types.TypeTraffic:
-		// Traffic type packets are forwarded normally by falling through. There
+		// Traffic type packets are forwarded normally by falling through unless hop
```

Comment:
Guessing this is accidental 😄

Reply:
Haha yeah I must have missed this in a merge. It interleaved the new and old comments.